packages/server/api/routes/reviews.router.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const reviewsController = require('../controllers/reviews.controller');
+
+/**
+ * @swagger
+ * /reviews:
+ *  get:
+ *    tags:
+ *    - reviews
+ *    summary: Get all reviews
+ *    description:
+ *      Will return all reviews.
+ *    produces: application/json
+ *    responses:
+ *      200:
+ *        description: Successful request
```

Comment:
Please describe the shape of the returned data. Is it a list? What properties do the items in that list have?